src/act.ts

```diff
@@ -375,7 +375,7 @@ export class Act {
             ...settings.options.map(option => option.path ? `--${option.name} ${option.path}` : `--${option.name}`)
         ];
 
-        const command = `${actCommand} ${Option.Json} ${commandArgs.options.join(' ')} ${userOptions.join(' ')}`;
+        const command = `${actCommand} ${Option.Json} ${Option.Verbose} ${commandArgs.options.join(' ')} ${userOptions.join(' ')}`;
```

Comment:
In terms of what is actually shown to the user, I filter out the debug and trace logs. Even with this filtering, are you noticing some excess messages?

If yes, how do you propose to still support skipped job/step status as I would still like to support Runner.Client.

Reply:
@ChristopherHX I went ahead with merging, but please let me know what you meant by it would spam the console. Since we do all the filtering, it should ideally only show the same messages as before unless the user purposely adds the `verbose` option from the Settings view.

Reply:
Great to hear!